modules/scripts/src/base/commands.ts

```diff
@@ -87,6 +105,10 @@
 				returns: "any",
 				parametersForm: "arguments",
 				handler: function(context: fugazi.app.modules.ModuleContext, index: string | number, value: any[] | fugazi.collections.Map<any>) {
+					if (fugazi.isPlainObject(value)) {
+						value = fugazi.collections.map(value);
+					}
```

Comment:
This is here because some functions pass in plain JavaScript objects that aren't Maps. For example http returns a `HttpResponse` which comes through as a plain object so we need to turn it into a `Map` so that it can we dealt with properly